src/main/java/org/jenkinsci/plugins/github_branch_source/GitHubSCMSource.java

```diff
@@ -353,7 +362,32 @@ public GitHubSCMSource(String repoOwner, String repository, String repositoryUrl
      */
     @Deprecated
```

Comment:
Is it really required to be deprecated or just not intended to be used for data-binding? It could be used programatically without problems.